variations/build-variations.mjs

```diff
@@ -32,8 +35,16 @@ async function buildVariation(source, variation) {
 	console.log( `Copying the source ${source} to the variation ${variation}` );
 
 	try {
+		// First grab the existing version if the variation exists already
+		let variationExists = fs.existsSync(`${destDir}/style.css`);
+		let currentVersion = null;
 
-		// First empty the old directory.
+		if( variationExists ) {
+			let styleCss = fs.readFileSync(`${destDir}/style.css`, 'utf8');
+			currentVersion = getThemeMetadata(styleCss, 'Version');
```

Comment:
Why do we pull the version number from the generated version instead of the source? 

For example, I'm looking at `variations/quadrat/quadrat-black/style.css` — the [version number in the variation](https://github.com/Automattic/themes/blob/trunk/variations/quadrat/quadrat-black/style.css#L10) lower than the [built version](https://github.com/Automattic/themes/blob/trunk/variations/quadrat/quadrat-black/style.css#L10). Is is that expected or a different issue?

Reply:
Discussed below.